src/components/ButtonWithTracking/index.tsx

```diff
@@ -1,10 +1,11 @@
 import React from 'react';
-import { Button, ButtonProps } from '@mui/material';
+import { Button, ButtonProps, Tooltip } from '@mui/material';
 import { useAnalyticsContext } from '../../contexts/AnalyticsContext';
 
 export interface ButtonWithTrackingProps extends ButtonProps {
 	eventName: string;
 	eventProperties?: { [key: string]: any };
+	tooltip?: string;
```

Comment:
Not a strong feeling about this, but perhaps we should not use a tooltip prop and instead just wrap any Button we want to have a tooltip with Tooltip manually? Just concerned of whether this is the beginning of creating complex custom components as in rendition (I can be overreacting lol)

Reply:
That's what I was initially doing, but i discovered that this can't be done without having to manipulate all our components. We should make all components wrapped in a forwardRef, which I honestly don't really like. So to make it work, we need to use Tooltip to only wrap MUI components.